textdb/textdb-gui/js/app.js

```diff
@@ -0,0 +1,34 @@
+var main = function(){
+		
+    $('.icon-menu').click(function(){
```

Comment:
Don't use anonymous functions, declare the function above as `var iconMenuClick = function() {}`